articles/what-is-jenkins/index.md

```diff
@@ -0,0 +1,61 @@
+
+### Why Continuous Integration is needed
+- Using CI developers tend to experience fewer conflicts when integrating code, this is because when code get merged it does not easily break. Also, even if it breaks the problem is easily resolved.
+- A developer may forget to run the project code before checking the changes, on the other hand, the CI server does not forget and the problem is caught early on and resolved.
+- CI supports automation through the process of building or compiling, testing, and reporting to the development team.
+- CI allows continuous delivery to [take place](https://medium.com/the-making-of-whereby/why-continuous-integration-is-so-important-7bb63ba5dc57).
+- Using CI locating and fixing bugs becomes very easy because developers do not have to wait too long to get the test results.
+- Developers are continuously notified about the build status, test results, and how the application is doing.
+
+### Continuous Integration with Jenkins
+Jenkins achieves continuous integration with the help of plugins. It has over a thousand plugins that integrate with every tool in the CI and CD (continuous delivery) toolchain. Jenkins reduces the development cycle since code gets to build and is tested continuously, thus releasing new features to the user faster and with fewer errors. A short time is taken to integrate the code, but before the use of Jenkins, integration was done manually and more time was required. The use of Jenkins enables integration to take place continuously. The use of Jenkins has helped the developers get feedback faster after the code has been integrated, this is helpful because if code breaks they know what the issue is and how it will be resolved.
+
+### Features of Jenkins
+ - Easy Configuration
+
+ Setting up Jenkins is easy and the web interface makes it is easy for developers to check for any errors, also they get great built-in help.
+
+ - Easy Installation
+
+ Jenkins is a self-contained Java program, it allows developers to run on the most popular operating systems such as Windows, Mac OS, and Linux.
+
+ - Plugins
+
+ Plugins are tools that make Jenkins more powerful for developers and continuous integration teams. Plugins allow Jenkins to integrate with development, testing, and deployment tools.
+
+ - Extensibility
+
+ Jenkins is extensible because the opportunity for developers to be able to configure them to fit with their environment is almost endless. The extensibility can be continued with newer releases of Jenkins, thus ensuring teams have the latest version of Jenkins.
+
+ - Distributed
+
+ Jenkins easily distributes work across multiple machines, helping in faster builds, tests, and deployments across multiple platforms.
```

Comment:
May be useful to do a quick conclusion summarizing your main points and include further resources.